api/authenticator.py

```diff
@@ -1528,15 +1667,18 @@ def remote_patron_lookup(self, access_token):
         """
         raise NotImplementedError()
     
-    def _internal_authenticate_url(self, library_short_name):
+    def _internal_authenticate_url(self, _db):
```

Comment:
This is how we get all the configuration settings when generating the authentication document. There used to be a single value that was passed in; now we get a database connection and look everything up.